Resources/Textures/_DV/Mobs/Customization/Thaven.rsi/meta.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "cheek_barbels, eyebrow_barbels, underbelly_face, underbelly_torso, carp_spots by kush.",
```

Comment:
`kushbreth (github)` and rename the rsi to be lowercase then update its uses

Reply:
getting trolled by windows case insensitivity